src/serverTest/serverTest.cpp

```diff
@@ -15,8 +17,12 @@ serverTest()
 	 *	AF_INET + SOCK_STREAM always protocol = 0,
 	 *	address for this socket, let the OS choose = INADDR_ANY
 	 */
-	const int				 MAXLINE = 1000;
-	Socket					 sock(AF_INET, 8080, SOCK_STREAM, 0);
+
+	json::Value val(config);
+	const int	MAXLINE = 1000;
+	std::cout << "HERE " << std::endl;
+	double					 port = val.get("port").get<double>();
```

Comment:
The port maximum is the same of the maximum port `~65000 ~= 2^16 - 1`
a short is 16 bit integer
```suggestion
	unsigned short					 port = val.get("port").get<double>();
```

Reply:
nice, thanks